### ISSUE TITLE
No code on duplicate enum member value

### ISSUE THREAD

**Issue by BOBJohnson** [dotnet/roslyn-analyzers#3605]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

v3.0.0 (Latest)

#### Diagnostic ID

No Diagnostic ID Is given when an enum has multiple members with the same value. The description in the error list is:
"The enum member '[member name]' has the same constant value '[member value]' as member '[other member name]'.

The "Code" column is blank.

#### Repro steps

1. Create an Enum.
2. Give 2 or more members of the Enum the same value.
3. Compile.

#### Expected behavior
A Code value provided.
#### Actual behavior
Code value is blank.

**Comment by mavasani** [comment]
@Evangelink

**Comment by Evangelink** [comment]
I don't repro the issue on a fresh console app project where I have added FxCop v3.0.0 and adding the following code snippet:

```csharp
public enum E
{
    A = 1,
    B = 1,
}
```

![image](https://user-images.githubusercontent.com/11340282/81192826-f4aeb200-8fba-11ea-8f8f-9bab245affc1.png)

**Comment by mavasani** [comment]
Verified no repro with v3.0.0. @BOBJohnson Feel free to re-open if you have a code snippet that shows the original issue.

**Comment by BOBJohnson** [comment]
Sorry didn't see your comments until just now.
Apparently this is a VB only issue.

Not really sure how to reopen the issue other than comment on it?

![NoCode](https://user-images.githubusercontent.com/5168739/81309776-4d308e80-9051-11ea-8acf-3abec4011d81.png)

**Comment by BOBJohnson** [comment]
Apparently this might be due to PEBKAC as well. See: #3609 
After uninstalling the VSIX package I didn't know I had installed, I just recompiled the text project, and am no longer getting the enum warning at all.

I still have FxCop 3.0.0 installed via NuGet on the test project.

So I don't know if that's an issue or not: VB doesn't give you a warning when an enum has two members with the same value. Should it give the warning? Or that something that is only applicable to C#?

**Comment by BOBJohnson** [comment]
Seems like I should maybe getting the warning even though it's VB: #3102

**Comment by BOBJohnson** [comment]
![NoWarning](https://user-images.githubusercontent.com/5168739/81314772-7f44ef00-9057-11ea-8ba1-1214b2bd8d27.png)

^ It appears the warning is showing up in the build output, just not the Error List UI?

**Comment by Evangelink** [comment]
@BOBJohnson I am in the same situation. I can see the warning in the output window but not in the source editor nor the error window...

**Comment by Evangelink** [comment]
@mavasani That's not the first time where there are some weird behaviors going on when mixing the nuget package and the VS extension. I had a corworker who faced AD0001 issues with messages saying impossible to find the analyzer when mixing the extension and the nuget package.

**Comment by Evangelink** [comment]
@BOBJohnson You should have a button to reopen the issue on the bottom of the page.

**Comment by BOBJohnson** [comment]
Apparently that button is only available when I close an issue I created. Seems if y'all close it for me, I can't re-open as I'm not a contributor. Which is fine - I don't think non-contributors like me should be able to re-open issues that were closed by y'all.

**Comment by Evangelink** [comment]
Ping @mavasani

**Comment by mavasani** [comment]
Please uninstall the VSIX extension "Microsoft CodeAnalysis 2019" when installing the NuGet package. We are soon planning to deprecate the extension: https://github.com/dotnet/roslyn-analyzers/issues/3314

**Comment by Evangelink** [comment]
@mavasani is there any additional action you want to take on this ticket?

**Comment by mavasani** [comment]
Sure, we can close this out.
